packages/paste-website/src/pages/components/display-pill-group/index.mdx

```diff
@@ -0,0 +1,232 @@
+import {Paragraph} from '@twilio-paste/paragraph';
+import {DisplayPill, DisplayPillGroup} from '@twilio-paste/display-pill-group';
+import Changelog from '@twilio-paste/display-pill-group/CHANGELOG.md';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {FormPillVsDisplayPillTable} from '../../../components/FormPillVsDisplayPillTable';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/display-pill-group"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/display-pill-group/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Display Pill Group"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/display-pill-group"
+  storybookUrl="/?path=/story/components-display-pill-group--basic"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{DisplayPill, DisplayPillGroup, CalendarIcon}}>
+  {`<DisplayPillGroup aria-label="Your favorite sports:">
+  <DisplayPill href="https://google.com">
+    <CalendarIcon decorative size="sizeIcon10" />
+    Tennis
+  </DisplayPill>
+  <DisplayPill>Football</DisplayPill>
+  <DisplayPill href="/">Baseball</DisplayPill>
+  <DisplayPill>Basketball</DisplayPill>
+  <DisplayPill>Soccer</DisplayPill>
+</DisplayPillGroup>`}
+</LivePreview>
+
+## Guidelines
+
+### About Display Pill Group
+
+<Paragraph>
+  {props.pageContext.frontmatter.description} Display Pills are static and should be used in situations where data is
+  not being modified.
+</Paragraph>
+
+Display Pills can link to other pages. This can useful when the entity the pill represents has it's own detail page within the application. For example, when building a user profile page a user might be assigned a number of permission based roles. Each role might have a details page explaining what it can do. You can use a collection of display pills, which link to each assigned role, to visually represent the current users assigned permissions, and learn more about what that enables them to do within the application.
```

Comment:
```suggestion
Display Pills can link to other pages. This can useful when the entity the pill represents has its own detail page within the application. For example, when building a user profile page, a user might be assigned a number of permission-based roles. You can use a group of Display Pills to visually represent the assigned roles and to link to a detail page for each assigned role learn more about what that enables them to do within the application.
```

This still took me a couple read throughs to understand what use case this example was describing. I'm not even exactly sure this is right because it seems like this is more of a Badge use case, where the roles are describing the user, rather than the pill being its own entity.

Regardless, it might help to have the coded example above reflect whatever use case is described here (if you keep it at all).

Reply:
Removed